include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/RoborazziPlugin.kt

```diff
@@ -338,9 +343,10 @@ abstract class RoborazziPlugin : Plugin<Project> {
             val isTaskPresent =
               isAnyTaskRun(isRecordRun, isVerifyRun, isVerifyAndRecordRun, isCompareRun)
             // Task properties
-            if (!isTaskPresent) {
+            if (!isTaskPresent && extension.taskType.orNull == RoborazziTaskType.None) {
               test.systemProperties.putAll(roborazziProperties)
             } else {
+              val taskType = extension.taskType.orNull
```

Comment:
How about using extensionTaskType as the variable name to clarify that this is the task type for the extension?

Reply:
Good idea, will do.